cmd/assetsvc/postgresql_utils_test.go

```diff
@@ -108,16 +108,16 @@ func Test_getChartFiles(t *testing.T) {
 	pgManager, mock, cleanup := getMockManager(t)
 	defer cleanup()
 
-	expectedFiles := models.ChartFiles{ID: "foo"}
+	expectedFiles := models.ChartFiles{ID: "fo/o"}
```

Comment:
it's generally better to write new test. If this test fails we don't know if it's because the basic functionality is broken or because of an error with the new logic related to `/` in the name.

Reply:
Absolutely right! I'm gonna add the new test for adding the slash case instead of squashing the old one. My bad!